lib/mozart_fetcher/component.ex

```diff
@@ -61,24 +61,24 @@ defmodule MozartFetcher.Component do
     %{index: component_index, id: id, status: 500, data: %{}}
   end
 
-  defp metric(id, 200) do
+  defp metric(id, _endpoint, 200) do
     ExMetrics.increment("success.component.process")
     ExMetrics.increment("success.component.process.#{id}.200")
   end
 
-  defp metric(id, status) when is_integer(status) do
+  defp metric(id, endpoint, status) when is_integer(status) do
     ExMetrics.increment("error.component.process")
     ExMetrics.increment("error.component.process.#{id}")
     ExMetrics.increment("error.component.process.#{id}.#{status}")
     ExMetrics.increment("error.component.process.#{status}")
-    Stump.log(:error, %{message: "Non-200 response. Got status:#{status} for component #{id}"})
+    Stump.log(:error, %{message: "Non-200 response. Got status:#{status} for component #{id}, endpoint: #{endpoint}"})
```

Comment:
Does this convert to JSON easily? Could we have this format as it will make it easier to search if they're separate fields:
```rb
{
  message: "Non-200 response",
  status: status,
  component: id,
  endpoint: endpoint
}
```

Reply:
this is marked as outdated, but `status` and `component` still aren't their own keys, they are still interpolated into the message. I think having them as Sam's suggestion would be neater